src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +78,10 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ThrowableProblem crateProblem(JSONException e) {
+        return Problem.valueOf(Response.Status.BAD_REQUEST, e.getMessage());
```

Comment:
Do you know what is inside e.getMessage() ? According to API Guidelines it's not really good idea to show internal error messages in API.